variables.tf

```diff
@@ -33,3 +33,11 @@ variable "role_writer_users"{
   description = "ROLE WRITER USERS"
 } 
 
+variable "loggly_token" {
+  description = "LOGGLY TOKEN"
+  sensitive = true
+}
+
+variable "log_level" {
+  description = "LOG LEVEL"

```

Comment:
Very minor, but it could be nice to add a default value here to save users always having to set this explicitly